datasource/management/commands/db_changes_fcu.py

```diff
@@ -0,0 +1,989 @@
+                2385514,
+                622037,
+                703039,
+                1084016,
+                741956,
+                593249,
+                1097061,
+                472531,
+                443773,
+                879989,
+                3305434,
+                274098,
+                2489449,
+                400084,
+                562683,
+                127475,
+                375780,
+                212586,
+                4254258,
+                134437,
+                448077,
+                446185,
+                616894,
+                392198,
+                814672,
+                874470,
+                3450754,
+                22048,
+                1083895,
+                994435,
+                1945247,
+                281890,
+                90337,
+                286073,
+                484897,
+                544335,
+                837840,
+                1100028,
+                306597,
+                1139103,
+                621384,
+                57992,
+                785688,
+                463593,
+                438395,
+                3593343,
+                4194765,
+                510693,
+                240084,
+                373795,
+                624787,
+                617882,
+                511579,
+                162980,
+                229584,
+                416197,
+                352183,
+                6486,
+                956880,
+                499994,
+                213293,
+                99189,
+                1086627,
+                311939,
+                608198,
+                856186,
+                1084829,
+                511832,
+                367075,
+                574088,
+                56799,
+                501132,
+                732637,
+                1107081,
+                628552,
+                150491,
+                940786,
+                1086056,
+                671996,
+            ]
+            try:
+                rssd = int(row["rssd"])
+                # checks if rssd number is in the list and adds _CDFI
+                if rssd in rssdIds:
+                    new_name = row[3] + "_CDFI"
+            except:
+                rssd = ""
+            # getting the datetime django wants
+            naive_datetime = datetime.now()
+            aware_datetime = timezone.make_aware(naive_datetime)
+            # removes special characters exept letters/numbers and spaces since we want to replace spaces with _
+            tag_name = re.sub(r"[^A-Za-z0-9 ]+", "", row["legal_name"].lower())
+            # creates the tag
+            tag = self._generate_tag(tag_name.lower().replace(" ", "_"), 0, existing_tags)
+            # adds tags to existing ones to check there are no duplicates
+            existing_tags.append(tag)
+            websiteurl = row["website"]
+            # checks if there is a website url
+            if websiteurl == None:
```

Comment:
again, define a function to return a correct websiteurl and call it. You shouldn't be calling brand.objects.create twice depending on this variable. It's overcomplicating the code